nodel-jyhost/src/main/java/org/nodel/jyhost/PyNode.java

```diff
@@ -568,7 +560,6 @@ private void applyConfig0(NodeConfig config) throws Exception {
             
             trackFunction("(instance creation)");
 
-            // _python = new PythonInterpreter(globals, pySystemState);
```

Comment:
It's strange this object gets built up with the correct working directories but never passed to the interpreter.